crytic_compile/platform/solc.py

```diff
@@ -521,15 +521,21 @@ def _run_solc(
     if not compiler_version.version in [f"0.4.{x}" for x in range(0, 11)]:
         # Add . as default allowed path
         if "--allow-paths" not in cmd:
-            relative_filepath = filename
-
-            if not working_dir:
-                working_dir = os.getcwd()
-
-            if relative_filepath.startswith(str(working_dir)):
-                relative_filepath = relative_filepath[len(str(working_dir)) + 1 :]
+            file_dir_start = os.path.normpath(os.path.dirname(filename))
+            file_dir = os.path.abspath(file_dir_start)
+            if file_dir.find(",") != -1:
```

Comment:
it might be clearer to write this as 
```suggestion
            if "," in file_dir:
```